src/openvic-simulation/map/TerrainType.cpp

```diff
@@ -1,4 +1,5 @@
 #include "TerrainType.hpp"
+#include <string_view>
```

Comment:
```suggestion
#include "TerrainType.hpp"

#include <string_view>
```
Include formatting (space between groups)